addons/website_slides/static/tests/tours/slides_tour_tools.js

```diff
@@ -87,6 +95,139 @@ var addArticleToSection = function (sectionName, pageName, backend) {
 }];
 };
 
+const fillInFileInput = async (input, name, type, content) => {
+    const blob = await fetch(`data:${type};base64,${content}`).then(res => { return res.blob(); });
+    const file = new File([blob], name, { type: type});
+    const dataTransfer = new DataTransfer();
+    dataTransfer.items.add(file);
+    input.files = dataTransfer.files;
+    input.dispatchEvent(new Event('change', { bubbles: true }));
+};
+
+const checkURLContent = async (url, name, type, expectedContent) => {
+	const blob = await fetch(url).then(res => { return res.blob(); });
+	const file = new File([blob], name, { type: type});
+	const actualContent = await utils.getDataURLFromFile(file).then(dataURL => {
+		return dataURL.split(',', 2)[1];
+	});
+	return expectedContent === actualContent;
+};
+
+const addImageToSection = function (sectionName, pageName, backend) {
```

Comment:
Also could add a small explanation of what you are testing here specifically (same for the pdf).